api/envoy/config/overload/v3/overload.proto

```diff
@@ -80,6 +82,31 @@ message Trigger {
   }
 }
 
+message ScaleTimersOverloadActionConfig {
+  enum TimerType {
+    // Adjusts the idle timer for downstream HTTP connections that takes effect when there are no active streams.
+    HTTP_DOWNSTREAM_CONNECTION_IDLE = 0;
+  }
+
+  message ScaleTimer {
+    // The type of timer this minimum applies to.
+    TimerType timer = 1;
+
+    oneof overload_adjust {
+      option (validate.required) = true;
+
+      // Sets the minimum duration as an absolute value.
+      google.protobuf.Duration min_timeout = 2;
+
+      // Sets the minimum duration as a percentage of the maximum value.
+      type.v3.Percent min_scale = 3;
+    }
+  }
+
+  // A set of timer scaling rules to be applied.
+  repeated ScaleTimer timer_scale_factors = 1 [(validate.rules).repeated = {min_items: 1}];
+}
```

Comment:
Please add release notes describing this new feature.

Reply:
Done.